ru-ru/nim-ru.html.markdown

```diff
@@ -169,89 +170,94 @@ var anotherArray = ["Default index", "starts at", "0"]
-  if raw_guess == "": continue # Пропустить эту итерацию
-  guess = str.parseInt(raw_guess)
-  if guess == 1001:
+  ввод_догадка: string
+  догадка: int
+
+while догадка != число:
+  ввод_догадка = readLine(stdin)
+
+  if ввод_догадка == "": continue  # Пропустить эту итерацию
+  
+  догадка = str.parseInt(ввод_догадка)
+
+  if догадка == 1001:
     echo("AAAAAAGGG!")
     break
-  elif guess > number:
-    echo("Nope. Too high.")
-  elif guess < number:
-    echo(guess, " is too low")
+  elif догадка > число:
+    echo("Неа. Слишком большое.")
+  elif догадка < число:
+    echo(догадка, " это слишком мало")
   else:
-    echo("Yeeeeeehaw!")
+    echo("Точнооооо!")
 
 #
 # Итерации (циклы)
 #
 
-for i, elem in ["Yes", "No", "Maybe so"]: # Или просто `for elem in`
-  echo(elem, " is at index: ", i)
+for i, элем in ["Да", "Нет", "Может быть"]:  # Или просто `for элем in`
+  echo(элем, " по индексу: ", i)
 
-for k, v in items(@[(person: "You", power: 100), (person: "Me", power: 9000)]):
-  echo v
+for ключ, знач in items(@[(чел: "You", сила: 100), (чел: "Me", сила: 9000)]):
+  echo знач
 
-let myString = """
-an <example>
-`string` to
-play with
+let мояСтрока = """
+<пример>
+`строки` для
+тренировки
 """ # Многострочная "сырая" строка
 
-for line in splitLines(myString):
-  echo(line)
+for строка in splitLines(мояСтрока):
+  echo(строка)
 
-for i, c in myString:       # Индекс и символ. Или `for j in` только для символов
-  if i mod 2 == 0: continue # Компактная форма `if`
-  elif c == 'X': break
-  else: echo(c)
+for i, симв in мояСтрока:     # Индекс и символ. Или `for j in` только для символов
+  if i mod 2 == 0: continue   # Компактная форма `if`
+  elif симв == 'X': break
+  else: echo(симв)
 
 #
 # Процедуры
 #
 
-type Answer = enum aYes, aNo
+type Ответ = enum оДа, оНет
 
-proc ask(question: string): Answer =
-  echo(question, " (y/n)")
+proc спрос(вопрос: string): Ответ =
+  echo(вопрос, " (д/н)")
   while true:
     case readLine(stdin)
-    of "y", "Y", "yes", "Yes":
-      return Answer.aYes  # Перечисления могут быть квалифицированы
-    of "n", "N", "no", "No":
-      return Answer.aNo
-    else: echo("Please be clear: yes or no")
-
-proc addSugar(amount: int = 2) = # Значение поумолчанию 2, ничего не возвращает
-  assert(amount > 0 and amount < 9000, "Crazy Sugar")
-  for a in 1..amount:
-    echo(a, " sugar...")
-
-case ask("Would you like sugar in your tea?")
-of aYes:
-  addSugar(3)
-of aNo:
-  echo "Oh do take a little!"
-  addSugar()
-# Здесь нет необходимости в `else`. Возможны только `yes` и `no`.
+    of "д", "Д", "да", "Да":
+      return Ответ.оДа  # Перечисления могут быть квалифицированы
+    of "н", "Н", "нет", "Нет":
+      return Ответ.оНет
+    else: echo("Поточнее, да или нет")
+
+proc добавьСахар(колво: int = 2) =   # Значение поумолчанию 2, ничего не возвращает
+  assert(колво > 0 and колво < 9000, "Диабет ☠")
+  for a in 1..колво:
+    echo(a, " кубик...")
+
+case спрос("Сахарку?")
+of оДа:
```

Comment:
I don’t know who maintained the repository, but if you don’t speak Russian, know that this is a crazy person, for example, there are sexual inclinations:

Reply:
In no case do not accept this request. For many Russian-speaking people, this text will cause a microstroke.

Reply:
Please explain what's wrong.